examples/summarization/train.py

```diff
@@ -190,14 +207,13 @@ def on_end_val_epoch(self, global_step, *collated_output, key="default"):
         refs = self.tokenizer.batch_decode(
             labels, skip_special_tokens=True, clean_up_tokenization_spaces=False
         )
-        # print(refs[:2])
-        # print(preds[:2])
+        logger.info(f"preds[:2]: {preds[:2]}")
```

Comment:
please comment out debug code if it's not necessary

Reply:
logger.debug() at least? log level will default to INFO anyways.